### PR TITLE
Update docker conf 2.0

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

**Comment by veolys-nanne** [comment]
:+1:

File: apache-php/vhosts/front-openorchestra-20.conf [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        Options -Indexes +FollowSymLinks -MultiViews
+        AllowOverride None
+        Order allow,deny
+        allow from all
+        DirectoryIndex app.php
+
+        <IfModule mod_rewrite.c>
+            RewriteEngine On
+
+                        RewriteRule ^sitemap.xml /None/sitemap.xml [L]
```
Comment (ngilain) [inline apache-php/vhosts/front-openorchestra-20.conf]: bad ident

File: apache-php/vhosts/media-openorchestra-20.conf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        Options -Indexes +FollowSymLinks -MultiViews
+        AllowOverride None
+        Order allow,deny
+        allow from all
+        DirectoryIndex app.php
+
+        <IfModule mod_rewrite.c>
+            RewriteEngine On
+
+
```
Comment (ngilain) [inline apache-php/vhosts/media-openorchestra-20.conf]: too many empty lines

**Comment by ngilain** [comment]
:+1:

**Pull request merged**